llvm/include/llvm/IR/ModuleSummaryIndex.h

```diff
@@ -472,14 +472,20 @@ class GlobalValueSummary {
     /// means the symbol was externally visible.
     unsigned CanAutoHide : 1;
 
+    /// This field is written by the ThinLTO indexing step to postlink
+    /// per-module summary. It indicates whether a global value (function or
```

Comment:
"postlink per-module" doesn't make sense - per-module is always pre-thinlink. Just say in the post link combined summary (which we do shard out for distributed ThinLTO, but it is still the "combined index" format).

Reply:
done.